libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -552,6 +557,14 @@ def bind_tools(
                     f"for the latest documentation on models that support tool choice."
                 )
             kwargs["tool_choice"] = _format_tool_choice(tool_choice)
+
+        if self._req_greedy_params():
+            kwargs["temperature"] = 1
+            kwargs["top_p"] = 1
+            kwargs["additional_model_request_fields"] = {} if self.additional_model_request_fields is None else self.additional_model_request_fields
+            inference_config = kwargs["additional_model_request_fields"].setdefault("inferenceConfig", {})
+            inference_config.update({"topK": 1})
```

Comment:
Can we remove any model specific runtime defaults. We can specify these in the samples or documentation.